.azure/azure-pipelines.perf.yml

```diff
@@ -3,19 +3,62 @@
 # This pipeline builds and runs MsQuic performance tests.
 #
 
-trigger: none
-pr: none
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
+    - release/*
```

Comment:
We need to double check, we might need to exclude release/1.1 and release/1.0 here. Otherwise they still might trigger this pipeline. Same thing for 2.0 until we backport it there.

Reply:
It doesn't matter.